src/action/platform_info.rs

```diff
@@ -0,0 +1,271 @@
+// Copyright 2020 Google LLC
+//
+// Use of this source code is governed by an MIT-style license that can be found
+// in the LICENSE file or at https://opensource.org/licenses/MIT.
+
+use sys_info::{linux_os_release, os_type, hostname};
+#[cfg(target_os = "linux")]
+use libc::{uname, utsname};
+use libc::c_char;
```

Comment:
Try to put platform-specific imports locally next to the places that actually need them.

Reply:
Ok, thanks! Fixed.